source/guide/3.2/guide-policies.md

```diff
@@ -196,6 +263,12 @@ groups:
               # that was previously configured
               workflow: autoheal
 
+              # Name of the service we want to short list (using regular expressions) and
+              # watch - every Diamond event has the service field set to some value.
+              # In our case, the ExampleCollector sends events with this value set to "example".
+              service:
```

Comment:
The location of service in this example is wrong. It is not a trigger parameter but rather a policy property